plugins/modules/powervm_lpar_instance.py

```diff
@@ -80,7 +81,7 @@
         choices: ['aix','linux','aix_linux','ibmi']
     state:
         description:
-            - C(present) creates a partition of specifed os_type, vm_name, proc and memory on specified system_name
+            - C(present) creates a partition of specifed os_type, vm_name, proc and dedicated memory on specified system_name
```

Comment:
Not here add it on line 23 as 
"Currently supports creation of partition (powervm instance) with only processor and memory settings in dedicated mode"

Reply:
Fixed

Reply:
Can you also modify the line 23 as asked?